includes/class-plugin.php

```diff
@@ -643,6 +645,82 @@ public function show_admin_notices() {
         }
     }
 
+    /**
+     * Display the admin bar menu item.
+     *
+     * @param \WP_Admin_Bar $wp_admin_bar
+     *
+     * @return void
+     */
+    public function render_admin_bar( $wp_admin_bar ) {
+        if ( defined( 'WP_REDIS_DISABLE_ADMINBAR' ) && WP_REDIS_DISABLE_ADMINBAR ) {
+            return;
+        }
+
+        if ( ! $this->current_user_can_manage_redis() ) {
+            return;
+        }
+
+        $wp_admin_bar->add_node(
+            [
+                'id'    => 'redis-cache',
+                'title' => __( 'Object Cache', 'redis-cache' ),
+            ]
+        );
+
+        if ( $this->get_redis_status() ) {
+            $wp_admin_bar->add_node(
+                [
+                    'parent' => 'redis-cache',
+                    'id' => 'redis-cache-flush',
+                    'title' => __( 'Flush Cache', 'redis-cache' ),
+                    'href' => $this->action_link( 'flush-cache' ),
+                ]
+            );
+        }
+
+        if ( Metrics::is_enabled() ) {
+            $wp_admin_bar->add_node(
+                [
+                    'parent' => 'redis-cache',
+                    'id' => 'redis-cache-metrics',
+                    'title' => __( 'Show Metrics', 'redis-cache' ),
```

Comment:
Should we make the "Settings"?

Reply:
To confirm. This should be renamed "Settings" and dump them here:

<img width="723" alt="CleanShot 2023-02-08 at 19 22 34@2x" src="https://user-images.githubusercontent.com/1422996/217631026-5043ef5d-ec80-45e9-b297-dc66f2ef7549.png">

Reply:
Yeah 👍